### ISSUE TITLE
Enable PaperWM under Ubuntu 20.04.4 LTS, GNOME 2.36.8, X11

### ISSUE THREAD

**Issue by forgottosave** [paperwm/PaperWM#411]
While trying to activate the extension an error occurs, not enabling the extension:
`-En PaperWM
Error occurred in tiling @enable:

space.clip.raise_top is not a function

finish@/home/user/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/tiling.js:1466:28
monitorsChanged@/home/user/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/tiling.js:1536:9
Spaces@/home/user/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/tiling.js:1379:14
enable@/home/user/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/tiling.js:2328:14
safeCall@/home/user/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/extension.js:59:52
run@/home/user/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/extension.js:44:14
enable@/home/user/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/extension.js:109:9
_callExtensionEnable@resource:///org/gnome/shell/ui/extensionSystem.js:167:32
_onEnabledExtensionsChanged/<@resource:///org/gnome/shell/ui/extensionSystem.js:499:18
_onEnabledExtensionsChanged@resource:///org/gnome/shell/ui/extensionSystem.js:498:11`

**Comment by smichel17** [comment]
I'm assuming you mean gnome 3.36.8 ;)

I believe this is a documentation issue— you need to use an older version of PaperWM on Gnome versions before 40. See https://github.com/paperwm/PaperWM/pull/417

tl;dr `git checkout 38.2` in the directory where you cloned the repo should get you up and running

**Comment by Jaeho211** [comment]
@smichel17 Thank you for your comment.
In my case, under 20.04.5 LTS, "gnome-shell-3.36" branch is worked.

**Comment by jtaala** [comment]
Closing was was resolved, plus #478 was implemented to specifically address the issue of a user attempting to use later branches (e.g. `develop`) when their version of gnome is not supported/meant for that branch (user will be notified on attempting to enable PaperWM).
